packages/CollabCodeEdit-Client.package/CCEClientHeartbeat.class/properties.json

```diff
@@ -6,12 +6,9 @@
 		 ],
 	"commentStamp" : "",
 	"instvars" : [
-		"editType",
-		"methodOrClassIdentifier",
-		"editContent",
-		"timestamp" ],
-	"name" : "CCEEdit",
+		"client" ],
+	"name" : "CCEClientHeartbeat",
```

Comment:
I find the name confusing. With `CCEClientHeartbeat` you think it is a single object which represents one Heartbeat that is always sent. However, the object is an engine that sends heartbeats all the time, and not itself a heartbeat. Maybe you find a better name, for example:
* CCEClientHeartbeatGenerator
* CCEClientHeartbeatEngine
... or whatever

Reply:
yeah agreed on that..